client/containers/Pub/PubHeader/ResponsiveHeaderButton.tsx

```diff
@@ -17,13 +17,9 @@ const ResponsiveHeaderButton = React.forwardRef((props: Props, ref) => {
 	const { labelPosition, outerLabel, showCaret = false, simpleLabel, ...sharedProps } = props;
 	const largeOnlyProps = { outerLabel, showCaret };
 	const smallOnlyProps = { labelPosition };
-
 	const { viewportWidth } = useViewport();
 
-	if (viewportWidth === null) {
-		return null;
-	}
-	if (viewportWidth > mobileViewportCutoff) {
+	if (viewportWidth === null || viewportWidth > mobileViewportCutoff) {
```

Comment:
This is the only relevant change. The width/height values returned by `useViewport` are `null` during SSR. Previously, we returned `null` from `ResponsiveHeaderButton` during SSR. With these changes, we'll render the `LargeHeaderButton` by default. On mobile, the buttons will appear large and then shrink down once the document hydrates. I've confirmed that the sticky banner works on both mobile and desktop viewport widths.